lib/msal-common/src/request/AuthorizationCodeRequest.ts

```diff
@@ -34,4 +34,5 @@ export type AuthorizationCodeRequest = BaseAuthRequest & {
     code: string;
     codeVerifier?: string;
     correlationId?: string;
+    redirectStartPage?: string;
```

Comment:
Why is this needed in this request obj?

Reply:
Was needed because this was the request object being used in `RedirectHandler.initiateAuthRequest()` where the start page is cached. I just moved the caching back one level to `acquireTokenRedirect` so this is no longer needed